pkg/util/memory/utils.go

```diff
@@ -225,3 +242,8 @@ func intoRatio(x float64) (zMilli int64) {
 	zMilli = int64(x * kilo)
 	return
 }
+
+// Prevent false sharing of cache lines.
+// The typical cache line size is 64 bytes on most architectures, including x86 and ARM.
+// `cpu.CacheLinePad`(128-byte) will be a good option for other platforms, such as Apple Silicon.
+type holder64Bytes struct{ _ [64]byte }
```

Comment:
How about using `cpu.CacheLinePad` instead?

Reply:
`cpu.CacheLinePad` chooses different values for different CPU architectures. For x86, it is 64.